webTest/js/script.js

```diff
@@ -0,0 +1,70 @@
+// function hideShow(){
+//
+//   if(cheker%2=0){
+//     var hider=document.querySelector("#hide");
+//     hider.style.display="none";
+//   }
+// }
+var cheker = 2;
+var clikeShowHide = document.querySelector("#showHide");
+var hider = document.querySelector(".showOne");
+clikeShowHide.addEventListener('click', function() {
+    if (cheker % 2 === 0) {
+        hider.style.display = hider.style.display + " none";
```

Comment:
The advice I gave you in the iCafe about this was actually not necessary. I told you to use 'replace' in case there were other attributes for 'display' but there can actually only be one attribute.

So hider.style.display="none" and "hider.style.display="flex" like you had would have been fine.

Sorry about the confusion :P